lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -62,7 +69,10 @@ public void removeLocationUpdates(LocationListener listener) {
 
     @Override
     public void removeLocationUpdates(PendingIntent callbackIntent) {
-        throw new RuntimeException("Sorry, not yet implemented");
+        intentToRequest.remove(callbackIntent);
+        if (intentToRequest.isEmpty()) {
```

Comment:
Should we check that `listenerToRequest` is null before setting the engine's request to null? And should we check that `intentToRequest` is null in L65? It feels slightly wrong to stop receiving location updates in the engine if there are still listeners but no pending requests (and vice versa).